package.json

```
@@ -20,7 +20,8 @@
     "lint": "eslint . --ext .ts  --ext .tsx",
     "lint:fix": "npm run lint -- --fix",
     "format:check": "prettier --check src/**",
-    "format:fix": "prettier --write src/**"
+    "format:fix": "prettier --write src/**",
+    "markdown-link-check": "markdown-link-check README.md"
```

```suggestion
    "markdown-link-check": "markdown-link-check README.md"
```

What's the need for only checking the README? Why not all docs?

---

Removed